packages/esm-patient-search-app/src/compact-patient-search/compact-patient-search.test.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { render, fireEvent, waitFor, screen } from '@testing-library/react';
+import CompactPatientSearchComponent from './compact-patient-search.component';
+
+jest.mock('@openmrs/esm-framework', () => ({
+  ...jest.requireActual('@openmrs/esm-framework'),
+  useSession: jest.fn(() => ({
+    ...jest.requireActual('@openmrs/esm-framework').useSession(),
+    sessionLocation: {
+      uuid: 'location-uuid',
+    },
+  })),
+  useConfig: jest.fn(() => ({
+    ...jest.requireActual('@openmrs/esm-framework').useConfig(),
+    search: {
+      patientResultUrl: '/patient/{{patientUuid}}/chart',
+    },
+  })),
+}));
+
+describe('CompactPatientSearchComponent', () => {
+  it('renders without crashing', () => {
+    render(<CompactPatientSearchComponent isSearchPage={true} initialSearchTerm="" />);
+    expect(screen.getByRole('searchbox')).toBeInTheDocument();
+  });
+
+  it('updates search term on input change', async () => {
+    render(<CompactPatientSearchComponent isSearchPage={true} initialSearchTerm="" />);
+    const searchInput: HTMLInputElement = screen.getByRole('searchbox');
+
+    fireEvent.change(searchInput, { target: { value: 'John' } });
+
+    await waitFor(() => {
+      expect(searchInput.value).toBe('John');
+    });
+  });
+
+  it('clears search term on clear button click', async () => {
+    render(<CompactPatientSearchComponent isSearchPage={true} initialSearchTerm="" />);
+    const searchInput: HTMLInputElement = screen.getByRole('searchbox');
+    const clearButton = screen.getByRole('button', { name: 'Clear' });
+
+    fireEvent.change(searchInput, { target: { value: 'John' } });
+    fireEvent.click(clearButton);
+
+    await waitFor(() => {
+      expect(searchInput.value).toBe('');
+    });
+  });
```

Comment:
These tests verify absolutely nothing. Please do not write tests like this, @ayush-AI . @anjula-sack please do not approve tests like this.